cpp/src/strings/convert/convert_datetime.cu

```diff
@@ -433,6 +433,19 @@ struct datetime_formatter {
   const int32_t* d_offsets;
   char* d_chars;
 
+  __device__ auto units_base(timestamp_units units) -> int64_t
+  {
+    switch (units) {
+      case timestamp_units::hours: return 24L;
+      case timestamp_units::minutes: return 1440L;   // 24*60
+      case timestamp_units::seconds: return 86400L;  // 24*60*60
+      case timestamp_units::ms: /**/ return 86400000L;
+      case timestamp_units::us: /**/ return 86400000000L;
+      case timestamp_units::ns: /**/ return 86400000000000L;
+    }
+    return 1L;  // days
+  }
+
   // divide timestamp integer into time components (year, month, day, etc)
   // TODO call the simt::std::chrono methods here instead when the are ready
```

Comment:
@codereport I am mentioning about this **TODO**. 
```suggestion
  // TODO call the simt::std::chrono methods here instead when they are ready
```

Reply:
call `std::chrono` methods.

Reply:
I could do this as part of duration type feature. It's not fully relevant to this PR.